bin/server_package/installation/post-uninstall.sh

```diff
@@ -1,15 +1,27 @@
 #!/bin/bash
 
+if [ "$1" = "purge" -a -e /usr/share/debconf/confmodule ]; then
```

Comment:
Such silently fails are very strange

Reply:
-a -e /usr/share/debconf/confmodule checks for existence of debconf package. And purge does not and must not fail.